cmd/agency.go

```diff
@@ -24,7 +24,6 @@ Parent command for starting and pinging agency
 }
 
 var agencyStartEnvs = map[string]string{
-	"apns-p12-file":            "APNS_P12_FILE",
```

Comment:
Remove FCLI_AGENCY_APNS_P12_FILE from deployment Dockerfile as well

Reply:
yes, I actually thought this but was so lazy that didn't want to touch those Docker related files. Good point, I'll fix this!